base_custom_info/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Base Custom Info
+================
+
+This module allows to create custom fields in models without altering model's
+structure.
+
+Installation
+============
+
+This module serves as a base for other modules that implement this behavior in
+concrete models.
+
+This means that you should **not** install this module by itself. If you found
```

Comment:
This module is a technical dependency and is be installed in parallel to other modules.